niyontwali/updated-portfolio-nextjs
# Ft bookmarks page

## Description of the PR

In this PR, I added the bookmark feature which will be a store for all the links that I visited. This will be public and the end users will be able to visit the page by loging in using their twitter, github or gmail account.

## Testing the functionalities in the PR

1.  Clone the repo
2.  Run `npm i` to install all package
3.  Check in the root directory a `.env sample` in order to include it in your `.env` file
4.  Run `npm run dev` to start your development server'
5.  Go to the site and try to login using your google/gmail account or any of the login method
6.  You can now visit the page
